chcemvediet/apps/accounts/migrations/0005_socialapp_remove_twitter_and_linkedin.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def forward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    SocialApp.objects.filter(pk__in=[3, 4]).delete()
+
+def backward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    twitter = SocialApp.objects.create(pk=3, provider=u'twitter', name=u'Twitter OAuth')
```

Comment:
Robustnejsie by bolo nechat DB nech ID priradi sama asi. Pri fixtures mame explicitne ID, pretoze tam sa vieme spolahnut, ze predtym bola DB prazdna.

Reply:
Preto som explicitne uviedol `pk`, pretoze to tak bolo aj vo fixtures.
:ok: Opravane.